apps/showcase/src/components/showcase/forms/forms-pres.component.ts

```diff
@@ -0,0 +1,158 @@
+import { AsyncPipe, CommonModule, formatDate } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@angular/core';
+import { ReactiveFormsModule, UntypedFormControl } from '@angular/forms';
+import { O3rComponent } from '@o3r/core';
+import { Localization } from '@o3r/localization';
+import { CustomFormValidation } from '@o3r/forms';
+import { CopyTextPresComponent, FormsEmergencyContactPresComponent, FormsPersonalInfoPresComponent } from '../../utilities';
+import { EmergencyContact, PersonalInfo } from './contracts';
+import { FormsPresTranslation, translations } from './forms-pres.translation';
+import { dateCustomValidator, formsPresValidatorGlobal } from './forms-pres.validators';
+
+@O3rComponent({ componentType: 'Component' })
+@Component({
+  selector: 'o3r-forms-pres',
+  standalone: true,
+  imports: [
+    AsyncPipe,
+    CommonModule,
+    CopyTextPresComponent,
+    FormsEmergencyContactPresComponent,
+    FormsPersonalInfoPresComponent,
+    ReactiveFormsModule
+  ],
+  templateUrl: './forms-pres.template.html',
+  styleUrl: './forms-pres.style.scss',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class FormsPresComponent {
+
+  /** Localization of the component */
+  @Input()
+  @Localization('./forms-pres.localization.json')
+  public translations: FormsPresTranslation;
+
+  /** The personal info form object model */
+  public personalInfo: PersonalInfo;
+  /** The emergency contact form object model */
+  public emergencyContact: EmergencyContact;
+
+  /** Form validators for personal info */
+  public personalInfoValidators: CustomFormValidation<PersonalInfo>;
+  /** Form validators for emergency contact */
+  public emergencyContactValidators: CustomFormValidation<EmergencyContact>;
+
+  /** The form control object bind to the personal info component */
+  public personalInfoFormControl: UntypedFormControl;
+  /** The form control object bind to the emergency contact component */
+  public emergencyContactFormControl: UntypedFormControl;
+
+  public submittedFormValue = '';
+
+  public firstSubmit = true;
+  public firstEmergencyContactFormSubmit = true;
+  public firstPersonalInfoFormSubmit = true;
+  private readonly forbiddenName = 'Test';
+
+  constructor() {
+    this.translations = translations;
```

Comment:
You don't need to do these in the constructor - you could directly set the value when declaring the properties